state-sync/diem-data-client/src/diemnet/tests.rs

```diff
@@ -381,7 +380,8 @@ async fn bad_peer_is_eventually_added_back() {
 
     // Advance time so the poller sends a data summary request.
     tokio::task::yield_now().await;
-    mock_time.advance_async(DATA_SUMMARY_POLL_INTERVAL).await;
+    const SUMMARY_POLL_INTERVAL: Duration = Duration::from_millis(1_000);
```

Comment:
I think it's common to only use all caps for global constants, otherwise it looks odd. Let's change this to:

`let summary_poll_interval = Duration::from_millis(1_000);`

Reply:
Done. change it to `let summary_poll_interval = Duration::from_millis(1_000);`